test/contact-forms.spec.js

```diff
@@ -188,4 +188,11 @@ describe('contact forms', () => {
       reported_date: now.valueOf()
     });
   });
+  it('should log error thrown during filling a contact form', async () => {
+    try {
+      await harness.fillContactCreateForm('household', ['Peter"s']);
```

Comment:
i like that you re-used an existing form here. is that not possible for the app-forms test? less to maintain if we don't add a new form

Reply:
It is not possible for 2 reasons.
- When attempting to use the same form for the app-forms test, an error occurs because it is located in the contact forms folder rather than the app forms folder.
- Currently, none of the existing app forms are suitable for testing this particular scenario.
@kennsippell